pantab/_readermodule.c

```diff
@@ -207,7 +200,6 @@ static PyObject *read_hyper_query(PyObject *Py_UNUSED(dummy), PyObject *args) {
         hyper_destroy_rowset_chunk(chunk);
     }
 
-    hyper_close_rowset(rowset);
```

Comment:
Does removing this not have any impact on the standard use case of still just reading a full table?

Reply:
The rowset is now opened and closed through the `with` statement in 

```
with connection.execute_query(query) as result:
```

Does that answer your question?

Reply:
Ah OK cool - thanks for clarifying